load/UpdateDBPTextFilesets.py

```diff
@@ -174,16 +185,16 @@ def getBibleVersesFromSqlite(self, filesetId, databasePath):
 			# NOTE: it is possible that verse value has a format equals to: number + string e.g. "2a".
```

Comment:
please remove the three comment lines, since they are no longer valid

Reply:
@bradflood they have been removed as you can see in the following lines:

https://github.com/faithcomesbyhearing/dbp-etl/pull/103/files#diff-38516f84a6c11ce60e5c2399f71fb8e10c8fffbc654c8e51082685269a35aa11R184